src/Sentry/Exceptions/SentryStackFrame.cs

```diff
@@ -160,7 +161,7 @@ public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger)
             writer.WriteStringIfNotWhiteSpace("context_line", ContextLine);
             writer.WriteBooleanIfNotNull("in_app", InApp);
             writer.WriteStringIfNotWhiteSpace("package", Package);
-            writer.WriteStringIfNotWhiteSpace("platform", Platform);
+            writer.WriteString("platform", Constants.Platform);
```

Comment:
@bruno-garcia @mattjohnsonpint if we always write `csharp` for the root event, can `"platform"` be omitted from stackframe and transaction?

Reply:
It always needs to be sent to sentry, for all events including both errors and transactions.  But on stack frames, it should only be included if that frame originates from another platform.  So, if we end up bundling the native SDKs at some point, the stack frames captured by those SDKs would be different.  Thus, this line should be reverted to as it was, and we should keep `StackFrame.Platform` null by default.